data_creation/data_creation.py

```diff
@@ -0,0 +1,357 @@
+import os
```

Comment:
this directory should move under `fluent_pose_synthesis` (meaning, `fluent_pose_synthesis/data`
then, file names that do something (for example this one) should use a verb: `create_data.py`